dealership/contracts.py

```diff
@@ -1,12 +1,50 @@
-class Contract(object):
-    pass
+from .customers import Customer, Employee
+from .vehicles import Car, Truck, Motorcycle
 
+class Contract(object):
+    CAR_INTEREST = 1.07
+    MOTORCYCLE_INTEREST = 1.03
+    TRUCK_INTEREST = 1.11
 
 class BuyContract(Contract):
     def __init__(self, vehicle, customer, monthly_payments):
-        pass
+        self.vehicle = vehicle
+        self.customer = customer
+        self.monthly_payments = monthly_payments
+
+    def get_interest_rate(self):
+        if isinstance(self.vehicle, Car):
+            return Contract.CAR_INTEREST
+        elif isinstance(self.vehicle, Motorcycle):
+            return Contract.MOTORCYCLE_INTEREST
+        elif isinstance(self.vehicle, Truck):
+            return Contract.TRUCK_INTEREST
 
+    def total_value(self):
+        self.Total_Value = self.vehicle.sale_price() + (self.get_interest_rate() * self.monthly_payments * self.vehicle.sale_price()/100) # update the interest calling
```

Comment:
In python, variables and method names are usually all lowercase.

Reply:
will follow the convention

Reply:
Thank you